mailbox/opensearch/src/main/java/org/apache/james/mailbox/opensearch/OpenSearchMailboxConfiguration.java

```diff
@@ -36,46 +36,55 @@ public static class Builder {
         private Optional<IndexAttachments> indexAttachment;
         private Optional<IndexHeaders> indexHeaders;
 
+        private Optional<Boolean> messageIndexNewBehavior;
```

Comment:
Clear today, but this won't be clear tomorrow, that is for sure!

Instead describing what this new behaviour is - or tries to achieve -might be a better idea!

Eg of better idea: optimiseMoves